src/components/TokensTable.vue

```diff
@@ -20,6 +20,12 @@
             {{ tokensHints.smartContractId }}
           </hint-tooltip>
         </th>
+        <th>
+          Amount
```

Comment:
I'd prefer to have a more definitive header name e.g. "Minted Supply", "Mintable Supply" or "Initial Supply" depending on the meaning of this column (refer to my other comment

Reply:
Good question, I lost the original intention of this column